src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -26,6 +23,11 @@ public static class General {
 		public final static double VOLTAGE_COMP_VAL = 11.5;
 		public final static double RAMP_RATE_VAL = 0.4;
 		
+		public static class Field{
```

Comment:
Raz - Field data is not really a part of the robot map, maybe it should be in a diffrent file

Reply:
done